tests/components/xiaomi/test_device_tracker.py

```diff
@@ -185,7 +186,7 @@ async def test_config_full(xiaomi_mock, hass):
     }
     xiaomi.get_scanner(hass, config)
     assert xiaomi_mock.call_count == 1
-    assert xiaomi_mock.call_args == mock.call(config[DOMAIN])
+    assert xiaomi_mock.call_args == call(config[DOMAIN])
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5451397)